docs/product/tracing/index.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Tracing
+sidebar_order: 20
+description: "Learn about tracing and distributed tracing and how it can be used to not just observe, but also debug application performance issues."
```

Comment:
```suggestion
description: "Learn about tracing and distributed tracing and how it can be a powerful tool for not just observing, but also debugging application performance issues."
```